backend.js

```diff
@@ -96,13 +122,26 @@ Backend.prototype.getTile = function(z, x, y, callback) {
             return source.getTile(bz, bx, by, sourceGet);
         } else if (typeof backend._lookback === 'number' &&
             err && err.message === 'Tile does not exist' &&
-            lookbacks <= backend._lookback) {
-            lookbacks += 1
-            bz = bz - 1;
-            bx = Math.floor(x / Math.pow(2, z - bz));
-            by = Math.floor(y / Math.pow(2, z - bz));
+            lookbacks === true) {
+            lookbacks == false;
+            var q = new queue.queue();
             headers['x-vector-backend-object'] = 'fillzoom';
-            return source.getTile(bz, bx, by, sourceGet);
+            for (var lb = 1; lb <= backend._lookback; lb++) {
+                bz-= 1
+                bx = Math.floor(x / Math.pow(2, z - bz));
+                by = Math.floor(y / Math.pow(2, z - bz));
+                q.defer(loadAsync, bz, bx, by);
+            }
+            return q.awaitAll(function(err, data) {
+                data = data.filter(function(d, i) {
+                    return d === null || i === 0;
+                });
+                bz = data[0].z;
+                bx = data[0].x;
+                by = data[0].y;
+                source.getTile(bz, bx, by, sourceGet);
```

Comment:
@yhahn - Trying to work out a way to not use `source.getTile` again, and use what we've already loaded here: https://github.com/mapbox/tilelive-vector/pull/131/files#diff-6d11f931df6358ebec2a997c99b90d66R95